lib/index.js

```diff
@@ -44,6 +44,11 @@ class ElectronDownloader {
   }
 
   get cache () {
+    const oldCacheDirectory = path.join(os.homedir(), './.electron')
+    if (pathExists.sync(path.join(oldCacheDirectory, this.filename))) {
+      return path.join(os.homedir(), './.electron')
+    }
+
     // use passed argument or XDG environment variable fallback to OS default
     return this.opts.cache || envPaths('electron', {suffix: ''}).cache
```

Comment:
Shouldn't this method always return `this.opts.cache` when specified? Even if the old directory exists.